src/kudu/consensus/consensus_meta.h

```diff
@@ -275,6 +285,11 @@ class ConsensusMetadata : public RefCountedThreadSafe<ConsensusMetadata> {
   // used to populate this value.
   std::atomic<int64_t> on_disk_size_;
 
+  // Tracks the last 'max_removed_peers' peers that have been removed
+  // from the config
+  static const int max_removed_peers = 30;
```

Comment:
30 might be too much. 10 should be good enough?

Reply:
SInce we are only storing uuid, 30 does not seem like a lot. Seems fine to me. WDYT?